library/core/query/src/main/kotlin/de.lise.fluxflow.query/filter/Filter.kt

```diff
@@ -152,6 +152,18 @@ interface Filter<in TModel> {
             return ContainsElementFilter(element)
         }
 
+        /**
+         * Creates a filter that matches collections not containing the given [element].
+         * @param element The element to be absent within matching collections.
+         * @return A filter matching all collections not containing the given [element].
+         */
+        @JvmStatic
+        fun <TCollection : Collection<TModel>, TModel> notContainsElement(
```

Comment:
I suggest renaming `containsNot` to `doesNotContain` for better clarity and consistency with our existing contains filter